tests/test_admin.py

```diff
@@ -36,3 +49,90 @@ def test_state_label(self, admin):
         log = TriggerLog(id=0, table_name='TABLE', record_id=100,
                          action=TRIGGER_LOG_ACTION['INSERT'])
         assert log.get_state_display() in admin.state_label(log)
+
+
+@pytest.mark.django_db
+class TestAdminActions:
+
+    @pytest.fixture
+    def mock_admin(self):
+        return mock.create_autospec(admin.TriggerLogAdmin)
+
+    @pytest.fixture
+    def mock_request(self):
```

Comment:
Django has the `RequestFrancotry` and `ptest-django` has the `rf` fixture, which does the same thing for you :)

Reply:
Anyhow, since you are using the database for all tests here, why not just go with `admin_client`, another pytest-django fixtures, that is a client (subclass of RequestFactory) that has a superuser session.